tests/test_dfes.py

```diff
@@ -5,10 +5,219 @@
 import pytest
 import stdpopsim
 import textwrap
+import numpy as np
 from stdpopsim import dfe
 from stdpopsim import utils
 
 
+class TestCreateMutationType:
+    """
+    Tests for creating a MutationType instance.
+    """
+
+    def test_default_mutation_type(self):
+        mt = dfe.MutationType()
+        assert mt.dominance_coeff == 0.5
+        assert mt.distribution_type == "f"
+        assert mt.distribution_args == [0]
+        assert mt.convert_to_substitution == True
+
+    def test_Q_scaled_index(self):
+        mt = dfe.MutationType(
+                 distribution_type="f",
+             )
+        assert mt.Q_scaled_index == [0]
+
+        mt = dfe.MutationType(
+                 distribution_type="e",
+                 distribution_args=[1],
+             )
+        assert mt.Q_scaled_index == [0]
+
+        mt = dfe.MutationType(
+                 distribution_type="g",
+                 distribution_args=[0.014, 0.19],
+             )
+        assert mt.Q_scaled_index == [0]
+
+        mt = dfe.MutationType(
+                 distribution_type="n",
+                 distribution_args=[0.5, 1],
+             )
+        assert mt.Q_scaled_index == [0, 1]
+
+        mt = dfe.MutationType(
+                 distribution_type="w",
+                 distribution_args=[0.5, 1],
+             )
+        assert mt.Q_scaled_index == [0]
+
+        mt = dfe.MutationType(
+                 distribution_type="l",
+                 distribution_args=[0.5, 1],
+             )
+        assert mt.Q_scaled_index == []
+
+    def test_create_bad_mutation_type(self):
```

Comment:
These are great, and no need to change these, but FYI, a nicer way to do this is like
```
with pytest.raises(ValueError, match="dominance_coeff"):
    mt = dfe.MutationType(dominance_coeff=True)
```

The code is a bit shorter, and I think it's nice to test on a good substring rather than exact equality of the error string (since what if we do minor edits to the error string in the future).